MDNetworking/Wrappers/MDListDataConverters.cs

```diff
@@ -92,40 +93,45 @@ public object[] ConvertForSending(object Item)
             }
 
             List<object> ObjectArray = new List<object>();
+            List<object> newLastValues = new List<object>();
 
-            LastValues.Clear();
-            foreach (MemberInfo info in Members)
+            for (int i = 0; i < Members.Count; i++)
             {
-                object value = info.GetValue(Item);
-                LastValues.Add(value);
-                ObjectArray.Add(value);
+                object value = Members[i].GetValue(Item);
+                if (Complete || LastValues.Count == 0 || Equals(LastValues[i], value) == false)
+                {
+                    ObjectArray.Add(i);
+                    ObjectArray.Add(value);
+                }
+                newLastValues.Add(value);
             }
+
+            LastValues = newLastValues;
             return ObjectArray.ToArray();
         }
 
         public object CovertBackToObject(object CurrentObject, object[] Parameters)
         {
             ExtractMembers();
-            if (Parameters.Length < Members.Count)
-            {
-                return null;
-            }
 
             T obj;
             if (CurrentObject != null)
             {
                 // Replace values in existing object
+                GD.Print("We got one");
```

Comment:
These GD.Prints should be MDLog.Trace/Debug

Reply:
Ops was leftover from testing, will remove